src/makeInfinityAdder.js

```diff
@@ -4,7 +4,23 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  function adder(a) {
+    if (a !== undefined) {
```

Comment:
You are checking for `undefined` to terminate the addition sequence. This works, but it might be clearer to check explicitly for the absence of arguments using `arguments.length === 0`.